src/main/connector/system/Linux/Linux.yaml

```diff
@@ -58,30 +58,62 @@ monitors:
           system.cpu.time{system.cpu.state="system"}: $4
           system.cpu.time{system.cpu.state="idle"}: $5
           system.cpu.time{system.cpu.state="io_wait"}: $6
+  process:
+    simple:
+      sources:
+        procrb:
+          type: osCommand  
+          commandLine: (cat /proc/stat)
+          computes:
+          - type: awk
+            script: |
+             /procs_/ {s = s ";" $2} END {print s}
+      mapping:
+        source: ${source::procrb}
+        attributes:
+          id: $1
+          name: $2
+        metrics:
+          system.cpu.process{system.cpu.process="running"}: $2
+          system.cpu.process{system.cpu.process="blocked"}: $3           
   memory:
     simple:
       sources:
+        memorySys:
+          # memSystem
+          type: osCommand  
+          commandLine: (/usr/bin/lsmem -b) || (free -b | awk '/Mem/ {print "Total online memory " $2}')
+          computes:
+          - type: awk
+            script: |
+              /Total online memory/ { print $4 }
         memoryInfo:
           # memTotal;memFree;memUsed;memBuffers;memCached;memFreeUtilization;memUsedUtilization;memBuffersUtilization;memCachedUtilization
           type: osCommand
-          commandLine: /usr/bin/cat /proc/meminfo
+          commandLine: cat /proc/meminfo
           computes:
           - type: awk
             script: ${file::memory.awk}
+          - type: leftConcat
+            column: 1
+            value: ;
+          - type: rightConcat
+            column: 1
+            value: ${source::memorySys}
       mapping:
         source: ${source::memoryInfo}
         attributes:
           id: memory_usage

```

Comment:
Is this `id` required?

@NassimBtk There will be cases where we don't want an `id` attribute, because there is only 1 instance.

Idea: we could consider that if `id` attribute is not set, then there is a single instance and it's "internal ID" is something like "_singleton" or just the name of the monitor type (in this example: `memory`?)

Reply:
Using the host.name and host.id and maybe other attributes the metric is unique so the id is not required, yes. Currently, the engine requires an id for the internal collect step otherwise it skip the monitor instance...
Indeed, the engine can be enhanced to determine an id under the hood.

Reply:
New issue to be created in metricshub ;).

Reply:
https://github.com/sentrysoftware/metricshub/issues/160